articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+    ![Azure site recovery installation 6](images/azs-siterecovery-installer-step5.png)
+
+7. In **MySQL Configuration**, create credentials for logging on to the MySQL server instance that is being installed.
+
+    ![Azure site recovery installation 7](images/azs-siterecovery-installer-step6.png)
+
+8. In **Environment Details**, select **No**.
+
+    ![Azure site recovery installation 8](images/azs-siterecovery-installer-step7.png)
+
+9. In **Install Location**, select where you want to install the binaries and store the cache. It is recommended to install the unified setup into the cache drive you created earlier as it has at least 600gb.
+
+    ![Azure site recovery installation 9](images/azs-siterecovery-installer-step8.png)
+
+10. In **Network Selection**, specify the listener (network adapter and SSL port) on which the configuration server sends and receives replication data. Port 9443 is the default port used for sending and receiving replication traffic and this is the port we opened up on the NSG earlier, so in this example this is what we are using. **Do not use port 443 for this.**
+
+    ![Azure site recovery installation 10](images/azs-siterecovery-installer-step9.png)
+
+11. In **Summary**, review the information and select Install.
+
+    ![Azure site recovery installation 11](images/azs-siterecovery-installer-step10.png)
+
+    ![Azure site recovery installation 12](images/azs-siterecovery-installer-step11.png)
+
+After registration finishes, the server should become option when selecting **Configuration Server**.
+
+## Step 4. Set up target environment
+
+In Public Azure, select and verify target resources.
+
+1. In **Prepare infrastructure** > **Target**, select the Azure subscription you want to use.
+
+2. Specify the target deployment model.
+
+3. Site Recovery will check to see if you have a compatible Public Azure Storage account and network. If you don't you will need to create them. This can be done with the Storage **Account** and **Network** buttons under Target.
+
+4. Select **OK**.
+
+    ![Azure site recovery infrastructure 1](images/azs-browser-recovery-infrastructure-configuration3.png)
+
+## Step 5: Enable replication
+
+1. Select the final tab **Replication Settings**.
+
+2. Select **Create and Associate** and specify a policy name.
+
+3. In **RPO threshold**, specify the recovery point objective (RPO) limit.
+
+    * This setting does not affect replication, which is continuous. It simply issues an alert if the threshold limit is reached without a recovery point being created.
+
+4. In **Recovery point retention**, Specify how long each recovery point is kept. Replicated VMs can be recovered to any point in the specified time window.
```

Comment:
You can recover replicated VMs to any point...

Reply:
done